src/components/layout/main/section-header.tsx

```diff
@@ -0,0 +1,11 @@
+// TODO: add breadcrumbs
```

Comment:
```suggestion
```
remove unused comment

Reply:
It's important to remind me that we need some breadcrumbs in the near future.